src/sniper-locator-controller/src/main/kotlin/org/icpclive/sniper/Application.kt

```diff
@@ -66,47 +54,24 @@ private fun Application.setupKtorPlugins() {
 
 @Suppress("unused") // application.yaml references the main function. This annotation prevents the IDE from marking it as unused.
 fun Application.module() {
-    LocatorController.overlayUrl = environment.config.property("live.overlayUrl").getString()
+//    environment.log.info("Using config directory ${Config.configDirectory.toAbsolutePath()}")
+//    environment.log.info("Current working directory is ${Paths.get("").toAbsolutePath()}")
     setupKtorPlugins()
-
+    Util.initForServer()
     routing {
         singlePageApplication {
```

Comment:
Do we actually need to host whole overlayer admin here? That sounds suspicious, most of it would not work anyway.